test/gsb-test-srv/proto/safebrowsing.proto

```diff
@@ -0,0 +1,467 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
This is a copy of some upstream file, right? Can you link to the original source for reference? Also, why do we need a local copy of the .proto and .pb.go? Does upstream not expose the types?